storybook/src/components/LinkList/LinkList.stories.tsx

```diff
@@ -6,40 +6,63 @@
 import { LinkList } from '@amsterdam/design-system-react/src'
 import * as Icons from '@amsterdam/design-system-react-icons'
 import { Meta, StoryObj } from '@storybook/react'
+import { contrastColorDecorator, inverseColorDecorator } from '../shared/decorators'
 import { exampleLinkList } from '../shared/exampleContent'
 
-const links = exampleLinkList()
+const linkList = exampleLinkList()
 
 const meta = {
   title: 'Components/Navigation/Link List',
   component: LinkList,
+} satisfies Meta<typeof LinkList>
+
+export default meta
+
+const linkMeta = {
+  component: LinkList.Link,
+  args: {
+    children: linkList[0],
+    contrastColor: undefined,
```

Comment:
Is this necessary? You're already defining these controls in the `LinkStoryTemplate`, no? Same goes for `inverseColor` and `size`